aiida/tools/visualization/graphviz.py

```diff
@@ -202,8 +202,7 @@ def draw_link_settings(inp_id, out_id, link_label, link_type):
     try:
         exit_code = subprocess.call(['dot', '-T', image_format, fname, '-o', output_file_name])
     except OSError:
-        from aiida.cmdline.utils import echo
-        echo.echo_critical('Operating system error - perhaps Graphviz is not installed?')
+        raise OSError('Operating system error - perhaps graphviz is not installed?')
```

Comment:
Maybe remove "Operating system error - " that seems a blue screen of death problem? :-)

Reply:
done